src/appresolver.js

```diff
@@ -42,6 +42,7 @@ function LocalAppRegistry(appClass, opts) {
 	opts.port = opts.port || 3000;
 	opts.dist = opts.dist || 'dist';
 	opts.configFile = opts.configFile || 'appconfig.json';
+	opts.useAppRoute = opts.useAppRoute || 'true';
```

Comment:
I think we want to keep this a boolean if we can... `yargs` definitely supports boolean flag arguments. But you may need to invert it so that it's `false` by default... so like `excludeAppRoute` maybe?

Reply:
Sounds good, I'll give that a try!

Reply:
So testing locally with the `frauLocalAppResolver` config in package.json is producing the following:
![image](https://user-images.githubusercontent.com/64804046/118021270-0d1d7f00-b329-11eb-911b-414ad0aecd8d.png)

So looks like it is passed as a string here 🤔

Reply:
@dlockhart should we just leave it a string then?

Reply:
I see. Yeah so you could leave it as a string, or turn it into a way to actually specify the path itself which defaults to `app/`.

Reply:
Hmm yeah, making it an option to specify the path will increase flexibility so I'll do that!